importer/helpers/dagbuilder.go

```diff
@@ -208,10 +231,15 @@ func (db *DagBuilderHelper) NewLeafDataNode(fsNodeType pb.Data_DataType) (node i
 	if err != nil {
 		return nil, 0, err
 	}
-	dataSize = uint64(len(fileData))
+	finalData := fileData
+	if db.tokenMetaToProcess {
+		finalData = append(fileData, db.tokenMetadata...)
```

Comment:
So, for the current design, we are not going to prepend? Or is `fileData` another kind of meta info?

Reply:
Additional question is why can't we prepend token metadata before splitting by chunker.

Reply:
1. Currently we append chunk metadata. fileData is a chunk with user data (from the file).
2. Additional question <- Chunker allocate buffer and read user file. So changing this behavior seemed 
  a little too much changes at Chunker to me.  If this is the way to go (like providing a new interface method 
  to have an argument with prepending data) it is not impossible. Or as of now we can proceed with the current 
  appending design and complete the whole and then modify later.